services/webhook/tests/test_async_dispatch.py

```
@@ -103,14 +115,75 @@ def fake_job(event):
+        ok = ad.enqueue_elder_review(
+            payload=payload, delivery_id="draft-delivery", blocking=False,
+        )
+
+    assert ok is True
+    enqueue.assert_not_called()
+
+
+def test_durable_elder_enqueue_failure_is_recorded_for_replay(monkeypatch):
+    monkeypatch.setenv("GRUG_ELDER_DURABLE_QUEUE", "1")
+    with patch("rerun.enqueue_review", side_effect=RuntimeError("queue down")):
+        with pytest.raises(RuntimeError, match="durable Elder review enqueue failed"):
```

Ruff (0.15.20)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)